dvc/scm/git/__init__.py

```diff
@@ -290,8 +290,8 @@ def get_diff_trees(self, a_ref, b_ref=None):
         """Method for getting two repo trees between two git tag commits.
         Returns the dvc hash names of changed file/directory
 
-        Args: a_ref(str) - git reference b_ref(str) - optional second git
-            reference, default None
+        Args: a_ref(str) - git reference
```

Comment:
still with this one ^ :sweat_smile:

Reply:
Woops... Didn't get that right. Fixed now I think.